### ISSUE TITLE
Add events for month and year changes

### ISSUE THREAD

**Issue by mostafajf** [icehaunter/vue3-datepicker#77]
we need an event when clicking month arrows without selecting any date. this is useful when we want to know the page on which we are

**Comment by UXandre** [comment]
Yes, been looking for the events for month and year changes here as well. Would love to see they are added!

**Comment by icehaunter** [comment]
Done & published as [0.3.7 on npm](https://www.npmjs.com/package/vue3-datepicker/v/0.3.7)
